### PR TITLE
chore: update testnet core image

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Dashmate was still deploying testnet with Core `20.0.0-beta.2`

## What was done?

Update testnet config to use default core version from base config and add migration

## How Has This Been Tested?

I don't know how to test this, incrementing version to 0.25.3 and running `yarn install && dashmate config` didn't work.

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [ ] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by strophy** [comment]
Seems like this is already done here?
https://github.com/dashpay/platform/pull/1483/files#diff-f98981e03a923c24fbe95372d2709a8cb5eab934f8cf0e1bc17147052c033c36R216-R218

**Review by shumkov — APPROVED** [review]
👍

File: packages/dashmate/configs/getConfigFileMigrationsFactory.js [hunk 1]
```diff
@@ -220,6 +220,16 @@ function getConfigFileMigrationsFactory(homeDir, defaultConfigs) {
 
         return configFile;
       },
+      '0.25.3': (configFile) => {
```
Comment (pshenmic) [inline packages/dashmate/configs/getConfigFileMigrationsFactory.js:223]: We already hit 0.25.3

**Review by pshenmic — CHANGES_REQUESTED** [review]

**Comment by pshenmic** [comment]
I think we can skip this PR, because all new users (new installs and upgrades) will get correct beta-4 version. Base config is set to correct version now, so it covers all the stuff, whether you are upgrading or startng new node.

**Pull request closed without merging**